modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/DartDioNextClientCodegen.java

```diff
@@ -46,6 +46,7 @@
     public static final String SERIALIZATION_LIBRARY_BUILT_VALUE = "built_value";
     public static final String SERIALIZATION_LIBRARY_DEFAULT = SERIALIZATION_LIBRARY_BUILT_VALUE;
 
+    private static final String DIO_IMPORT = "package:dio/dio.dart";
```

Comment:
when do you consider a string worth making `static final`?

Reply:
Good question, I don't really have a rule, when it gets complex enough to have typos I guess.

Reply:
just asking because there are other strings in this file that are duplicated more times than DIO_IMPORT ;)

Reply:
Ya I know :)